src/main/java/reposense/util/TimeUtil.java

```diff
@@ -137,7 +139,7 @@ public static void verifySinceDateIsValid(LocalDateTime sinceDate, LocalDateTime
     }
```

Comment:
The `verifySinceDateIsValid` function at line 134 should have the `currentDate` variable in the javadoc.